src/components/Pressure.vue

```diff
@@ -0,0 +1,348 @@
+<template>
```

Comment:
this should prob be in component lib, just like contention plot

Reply:
also probably neither is necessary for this example project, especially since this one has LCO specific stuff in it